vanilla-tests/src/test/java/fixtures/bodydatetime/DatetimeOperationsTests.java

```diff
@@ -82,14 +84,14 @@ public void putLocalPositiveOffsetMaxDateTime() {
     public void getLocalPositiveOffsetLowercaseMaxDateTime() {
         OffsetDateTime result = client.getDatetimeOperations().getLocalPositiveOffsetLowercaseMaxDateTime();
         OffsetDateTime expected = OffsetDateTime.parse("9999-12-31T09:59:59.999+00:00");
-        Assert.assertEquals(expected, result);
+        Assert.assertEquals(expected.toInstant().toEpochMilli(), result.toInstant().toEpochMilli());
```

Comment:
Any idea why now need to compare epoch?

Reply:
This is not directly related to the Iterable change but a side-effect of upgrading azure-core and azure-core-http-netty packages. The serializer versions were upgraded too and all these tests failed. The epoc millis were the same but the zone offset was different and the equality check failed.